client/plugin-src/cordova-plugin-ssh/ssh.js

```diff
@@ -0,0 +1,49 @@
+/* globals cordova, window */
+
+window.ssh = {};
+window.ssh._state = null;
+
+var sendCommand = function(method, params) {
```

Comment:
Not super important, but I have a slightly more concise version of the same thing here:
https://github.com/uProxy/colony/blob/ryscheng-openvpn/client/plugin-src/cordova-plugin-openvpn/openvpn.js